pebblo/app/models/models.py

```diff
@@ -126,6 +126,7 @@ class AiApp(BaseModel):
     lastUsed: datetime
     pebbloServerVersion: Optional[str]
     pebbloClientVersion: Optional[str]
+    clientVersion: Optional[dict]
```

Comment:
It seems to be a duplicate of the `framework`(L125), just with a different data type.

Reply:
Will use Framework info in clientVersion as 
`clientVersion: Optional[FrameworkInfo]`

And here `langchain_community_version` and its version will get stored